app/views/questionnaire.py

```diff
@@ -216,18 +211,20 @@ def get_timeout_continue(eq_id, form_type, collection_id):  # pylint: disable=un
     return 'true'
 
 
-@questionnaire_blueprint.route('session-expired', methods=["GET"])
+@questionnaire_blueprint.route('session-expired', methods=["POST"])
```

Comment:
Not sure the name of this make sense now; should it be an action like `sign-out` as that's what it's doing?